sty/etc/calbf.sty

```diff
@@ -38,7 +38,7 @@
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 % specal abbreviations for \bf and \cal math symbols
 
-\def\phi{\varphi} \def\tilde{\widetilde}
+\def\phi{\varphi} %\def\tilde{\widetilde}
```

Comment:
Christian talked about this to me during the summer.  When generating a Chinese module, there's always an error caused by this, even though the generated file is totally fine.